controllers/transactionsController.js

```diff
@@ -0,0 +1,29 @@
+const transactionsModel = require('../models/transaction')
```

Comment:
This file since it is already inside `controllers` folder, need not be called `transactionController`. Simply `transactions` would do.

Reply:
All existing controllers are following same naming convention.

Reply:
Oh ok...will change it